libs/blocks/modal/modal.js

```diff
@@ -3,7 +3,7 @@
 import { createTag, getMetadata, localizeLink, loadStyle, getConfig } from '../../utils/utils.js';
 import { decorateSectionAnalytics } from '../../martech/attributes.js';
 
-const FOCUSABLES = 'a:not(.hide-video), button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])';
+const FOCUSABLES = 'a:not(.hide-video), button:not([disabled], .paddle), input, textarea, select, details, [tabindex]:not([tabindex="-1"])';
```

Comment:
What do you think of making this select logic a little more specific, just so you're not disabling the focus for .paddles other modal use cases. `button:not([disabled], .locale-modal-v2 .paddle)`

Reply:
@vhargrave Updated!